lib/sepia/src/util.js

```diff
@@ -129,10 +130,7 @@ function substituteWithOpaqueKeys(text) {
   var substitutions = globalOptions.substitutions;
   for (var i=0; i<substitutions.length; i++) {
     var subst = substitutions[i];
-    var actualValue = subst.actualValueFn();
-    while (text.indexOf(actualValue) > -1) {
-      text = text.replace(actualValue, subst.opaqueKey);
-    }
+    text = text.replace(new RegExp(escapeRegex(subst.actualValueFn()), 'g'), subst.opaqueKey);
```

Comment:
Thanks, the split approach makes a lot of sense since the pattern matching isn't required.

I'll update the PR. 👍